### ISSUE TITLE
NVDA no longer works on machines without sound card, such as AppVeyor CI runner

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#16269]
### Steps to reproduce:
Try starting NVDA on a VM in AppVeyor CI.
### Actual behavior:
NVDA fails to star with the following in the log:
```
CRITICAL - __main__ (06:22:13.355) - MainThread (7092):
core failure
Traceback (most recent call last):
  File "nvda.pyw", line 415, in <module>
  File "core.pyc", line 684, in main
  File "audio\soundSplit.pyc", line 81, in initialize
  File "pycaw\utils.pyc", line 197, in GetAudioSessionManager
  File "pycaw\utils.pyc", line 177, in GetSpeakers
_ctypes.COMError: (-2147023728, 'Element not found.', (None, None, None, 0, None))

```
### Expected behavior:
NVDA should start
#### NVDA installed/portable/running from source:
N/A
#### NVDA version:
All after merge of PR #16071 
#### Windows version:
N/A
#### Name and version of other software in use when reproducing the issue:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Versions before #16071 don't have this problem
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
N/A
## Additional context:
This currently prevents new Alpha builds from being created. 
cc @mltony as this is caused by your PR.
Also @seanbudd for awareness as it looks like @mltony 's work was merged despite failing CI. Our release process explicitly mentions that only PR's where CI was successful can be merged, and it would be good to follow this standard practice. The fact that our system tests used to fail intermittently should cause them to be improved, rather than failures to be ignored.
